src/post/post.service.ts

```diff
@@ -341,4 +341,8 @@ export class PostService {
 
         return ok(post);
     }
+
+    async getMentionsForPost(post: Post) {
+        return this.postRepository.getMentionsForPost(post);
```

Comment:
We don't want the controllers or helpers to directly access the `postRepository` so added this function here.